src/core/utils/formatterUtils/formatterUtils.ts

```diff
@@ -94,9 +113,70 @@ class FormatterUtils {
         return formattedValue;
     };
 
-    private getDynamicOption = <TOptionValue extends string | number = number>(
-        value: number,
-        option?: DynamicOption<TOptionValue>,
+    formatDate = (value: DateTime | number | string | undefined, options: IFormatDateOptions = {}) => {
```

Comment:
🤔 Duration and Date are different entities, so it might be clearer to keep them separate. There might also be situations where we want to format the duration independently of the current time.
Then we would have two functions: `formatDate` and `formatDuration`.

Reply:
Good point, as far as I know we currently only display durations based on the current date. In such cases, we still format a date as a duration. Given that this involves changes on DX and on Notion documentation, I would suggest to open an improvement ticket to further discuss internally if we should split this function it two and have sort of two formats like `NumberFormat` & `DurationFormat`, what do you think?

Reply:
Yes, sure, sounds good 👍